video-app/src/app/library/add/add.component.ts

```diff
@@ -34,25 +45,41 @@ export class AddComponent {
   constructor(private readonly router: Router
     ,         private readonly youtubeService: YoutubeService
     ,         private readonly vimeoService: VimeoService
-    ,         private readonly streamingPlatform: StreamingPlatformService) { }
+    ,         private readonly streamingPlatform: StreamingPlatformService
+    ,         private readonly store: Store<{ videos: VideoState }>) {
```

Comment:
proponowałbym trzymać gdzieś ten interfejs i go importować bo na pewno jeszcze w kilku miejscach się przyda:)

Reply:
jak to zrobić?